### PR TITLE
metrics: unify metrics naming

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: store/tikv/client.go [hunk 1]
```diff
@@ -210,9 +210,8 @@ func (c *rpcClient) closeConns() {
 func (c *rpcClient) SendReq(ctx goctx.Context, addr string, req *tikvrpc.Request) (*tikvrpc.Response, error) {
 	start := time.Now()
 	reqType := req.Type.String()
-	storeID := strconv.FormatUint(req.Context.GetPeer().GetStoreId(), 10)
 	defer func() {
-		metrics.TiKVSendReqHistogram.WithLabelValues(reqType, storeID).Observe(time.Since(start).Seconds())
```
Comment (shenli) [inline store/tikv/client.go]: The store id is missing. @wentaoxu PTAL

**Comment by shenli** [comment]
Rest LGTM

**Comment by coocood** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**